taskvine/src/manager/vine_manager.c

```diff
@@ -5425,6 +5494,17 @@ int vine_tune(struct vine_manager *q, const char *name, double value)
 	} else if (!strcmp(name, "option-blocklist-slow-workers-timeout")) {
 		q->option_blocklist_slow_workers_timeout = MAX(0, value); /*todo: confirm 0 or 1*/
 
+	} else if (!strcmp(name, "watch-library-logfiles")) {
+		char *runtime_dir = xxstrdup(getenv("VINE_RUNTIME_INFO_DIR"));
```

Comment:
Please do the directory creation in `vine_runtime_directory_create` so we can keep track of the expected directories in one place.  (I realize the dir won't get used unless the feature is enabled.)